wire-ios-data-model/Source/Proteus/ProteusService.swift

```diff
@@ -59,7 +59,7 @@ public final class ProteusService: ProteusServiceInterface {
     ) throws {
         logger.info("establishing session from prekey")
 
-        guard let prekeyBytes = prekey.base64EncodedBytes else {
```

Comment:
This was a misleading name: the prekey here is a base 64 encoded string, and we want to convert it to bytes after decoding it from base 64.